### DISCUSSION TITLE
Align last slide to the left when slide with is smaller the slider width

### DISCUSSION THREAD

**Discussion by mrleblanc101** [nolimits4web/swiper#4780] [Q&A]
### Clear and concise description of the problem

Currently, if your slide width is smaller than the swiper width, when you get to the last slide, the right edge of the slide will be stuck to the right edge of the swiper.
<img width="339" alt="Capture d’écran, le 2021-07-20 à 11 02 13" src="https://user-images.githubusercontent.com/17392251/126347588-a3eac6cd-6ef5-4a2f-9629-ca163aef67c2.png">

### Suggested solution

When you reach the last slide, there should be an option so that the left edge of the last slide is aligned with the left edge of the swiper.
<img width="337" alt="Capture d’écran, le 2021-07-20 à 11 04 07" src="https://user-images.githubusercontent.com/17392251/126347900-4e471917-fa64-413c-8406-e1c482a91d77.png">

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Would you like to open a PR for this feature?

- [ ] I'm willing to open a PR

**Comment by FloLech** [comment]
We are facing the same issue at the moment. Totally support this feature request!

**Comment by GemN** [comment]
I have the same problem and adding an offset create a ghost pagination dot which is not ideal.

**Comment by xxvii** [comment]
I get a lot of request from clients who want the pagination to reflect the number of slides, not the number of swipes needed to see the last slide, and they don't mind the empty space on the right after the last slide.

On Flickity you can achieve this behavior with the cellAlign option.
doc: https://flickity.metafizzy.co/options.html#cellalign
example: https://codepen.io/desandro/pen/MYQwbx

**Comment by xxvii** [comment]
At the moment, the most flexible solution I found is to add a margin-right to the last slide equal to 100% minus the width of the last slide itself.

Something like:
```css
.swiper-slide {
  --slide-width: 25%;
  width: var(--slide-width);
}

.swiper-slide:last-child {
  margin-right: calc(100% - var(--slide-width));
}

**Comment by mrleblanc101** [comment]
Any news on this ? @nolimits4web 
This seem like a very basic feature.
Most swiper work like this.
I understand why Swiper doesn't, but shouldn't it be an option ?

**Comment by TomSmedley** [comment]
+1 I really would like this.

**Comment by gaetansenn** [comment]
any new about this ? Thank you

**Comment by gaetansenn** [comment]
I found this : https://codepen.io/YUUMU/pen/KKMaxOZ

**Comment by mofman** [comment]
+1

**Comment by mnlmaier** [comment]
Any news on this?

**Comment by mofman** [comment]
PR: https://github.com/nolimits4web/swiper/pull/8021
